src/algorithms/tracking/TrackerHitReconstruction.cc

```diff
@@ -85,6 +89,74 @@ std::unique_ptr<edm4eic::TrackerHitCollection> TrackerHitReconstruction::process
 
     }
 
+    //------------------------------------------------
+    // Example code of adding random noise hits
+    // Shujie Li, 08.2024
+    // use the first raw hits to obtain the volume ID of the detector system
+    // this also make sure the hit container has at least one hit
```

Comment:
Is this something that you really want or just a consequence of how you're getting the volume? Might it not be better to have a completely separate algorithm which constructs these hits from knowledge of the readout structure to get the volume rather than relying on their being a hit.